app/src/main/java/ch/epfl/sdp/blindly/location/AndroidLocationService.kt

```diff
@@ -117,8 +109,15 @@ class AndroidLocationService(private var context: Context) : LocationService {
             return "Location not found"
         }
 
+        fun getCurrentLocationStringFromLocation(context: Context, location: Location): String {
+            val latitude = location.latitude
+            val longitude = location.longitude
+            val geocoder = Geocoder(context)
+            return toAddress(geocoder, latitude, longitude)
+        }
+
         private fun toAddress(geocoder: Geocoder, latitude: Double, longitude: Double): String {
-            val address = geocoder.getFromLocation(latitude, longitude, 5)
+            val address = geocoder.getFromLocation(latitude, longitude, 2)
```

Comment:
what does the last parameter stand for ? precision ?

Reply:
It the amount of result you want, I had it set to 1 at the very beginning but it sometimes failed I think
But 5 seemed overkill.
Though I could replace it with a const instead